firebase-vscode/webviews/components/EmulatorPanel.tsx

```diff
@@ -186,14 +179,11 @@ export function EmulatorPanel({
       <h2>Launch the Emulator Suite</h2>
       {/* TODO(christhompson): Insert some education links or tooltips here. */}
       <Spacer size="xxlarge" />
-      Current project ID:
-      {/* TODO(christhompson): convert this into a demo- prefix checkbox or something. */}
-      <VSCodeTextField
-        disabled={true}
-        className="in-line"
-        value={emulatorUiSelections.projectId}
-        onChange={(event) => projectIdChanged(event)}
-      ></VSCodeTextField>
+      <span>
+        {"Current project ID: "}
+        {/* TODO(christhompson): convert this into a demo- prefix checkbox or something. */}
+        <kbd>{emulatorUiSelections.projectId}</kbd>
```

Comment:
what is kbd? can we avoid the box styling and just leave it as text (in the span).

Reply:
It's like `this`.
I thought it'd be looking better, but I don't mind removing it.

Reply:
Switched to bold. What do you think?
I want to differentiate it from the label. It feels odd to me otherwise

<img width="337" alt="Screenshot 2024-01-25 at 08 27 51" src="https://github.com/firebase/firebase-tools/assets/20165741/e28f5ee3-03cc-422f-b102-bdf2d546e0da">

Reply:
ye let's just keep that! I think you might be merging into the wrong branch.

Reply:
Ah yes, I started from the previous "revert" Pr to be able to work locally